content/projects/kauri/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Kauri Wallet"
+date: 2018-03-29T14:28:31+13:00
+lastmod: 2018-03-29T14:28:31+13:00
+draft: false
+author: "Paul Sanderson"
+description: "Kauri is an open-source, multi-currency wallet that will be the foundation for advanced features like mobile staking, merchant tools, atomic exchanges and many more great NavCoin features."
+resources: []
+categories: []
+filter: ["Wallets"]
+bg_image: "/images/icons/icon-wallet-purple-large.png"
+slug: ""
+type: "project"
+feature_image: ""
+show_title_text: false
+in_progress: true
+completed: false
+show_on_roadmap: true
+show_on_projects: true
+allow_click_through: true
+percent_complete: 10
+github_url: "https://github.com/Encrypt-S/kauri-wallet"
+project_url: ""
+reddit_url: ""
+twitter_url: ""
+card_color: "#6984D8"
+font_color: "#fff"
+weight: 0
+---
+{{< project_header
+    title="Kauri Wallet"
+    subhead=""
+    gradient=""
+    text="Kauri gives your crypto wallet the features you expect from your banking app like accounts and recurring payments but then so much more."
+    captionPadding="100px"
+    imgSrc="/images/icons/nav_purple-ecofriendly-large.png"
+    imgHeight="300px"
+    imgPadding="10px"
+    bgColor="#FFF"
+    fontColor="#333">}}
+{{< center_text_section
+    title="It’s open source"
+    bgColor="#42b6eb"
+    gradient=""
+    fontColor="#FFF">}}
+    <p>Transparency is an important pillar of decentralised software, especially when trusting an application to store your cryptocurrency. The Kauri Wallet will always be free and open source, so you can rest easy knowing the software used to secure your funds can be validated by anyone at anytime.</p>
+{{< /center_text_section >}}
+{{< project_features
+    fontColor="#333"
+    gradient="false"
+    feature1Img="/images/icons/nav_blue-wallet-large.png"
+    feature1Title="Multi-Currency"
+    feature1Txt="Kauri is designed to work with multiple blockchains to enable all the advanced NavCoin features Encrypt S is building for your entire portfolio."
```

Comment:
"Kauri is designed to work with multiple blockchains, enabling users of NavCoin to store not only their NAV but all their other cryptocurrencies in one place, and users of other cryptocurrencies the chance to bring NAV into their portfolios. "

?

Reply:
It's too long for the content block. I've kept it as it is. We can always improve or add to this content later.